config_testing.ini

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
I intended to only use it for the tests I was going to try to run, which would only have been on one board. Didn't want to change the base config file in case it was being used for more important things.